__TESTS_BUNDLE_SIZE__/bundleSizeTestCases.ts

```diff
@@ -22,8 +22,8 @@ const bundleSizeTestCases:ITestCase[] = [
     ]
   },
   {
-    name: 'Tests a transformable image with Resize and just',
-    sizeLimitInKB: 11,
+    name: 'Tests a transformable image with Resize and ajust',
```

Comment:
change to "Adjust"?

Reply:
Done, also capitzlized Border and Quality.